src/components/ProjectSearch.js

```diff
@@ -10,7 +10,9 @@ import { getProjects } from "../actions"
 
 const ProjectSearch = props => {
   const [searchString, setSearchString] = useState("")
-  const { getProjects, projects } = props
+  const { getProjects, projects, projectLength } = props
+  // console.log(projectLength)
+  // console.log(projects)
```

Comment:
Can we remove these commented out lines?

Reply:
Removed!